index.html

```diff
@@ -342,7 +332,7 @@ <h1>About <br>
           </div>
         </section>
         </main>
-        <section class="contact-me">
+        <section class="contact-me" id="contact">
           <footer>
             <div class="blue-rounded">
               <img src="assets/contact-form.svg" alt="Contact form background image">
```

Comment:
- Please work on your form so it's properly aligned and looks like that of the Figma design.
- Please add the button interaction for the `Get in touch` button
##### Figma design
![Screenshot from 2022-06-03 18-42-31](https://user-images.githubusercontent.com/58771507/171918330-4e5a45d7-454e-4543-bd8f-056465ded9d0.png)

##### Your design
![Screenshot from 2022-06-03 18-42-02](https://user-images.githubusercontent.com/58771507/171918375-8fa9296b-acb5-4884-8219-a9e3a07a836c.png)